Source/ApiTemplate/Source/ApiTemplate/ApiTemplate.csproj

```diff
@@ -2,13 +2,14 @@
   <!--#if (AuthoringMode)-->
   <PropertyGroup>
     <!-- Modify these constants to enable/disable a feature to debug the template. Note that this only affects the C#. -->
-    <DefineConstants>$(DefineConstants);TreatWarningsAsErrors;Kestrel;IIS;NGINX;Azure;ApplicationInsights;HealthCheck;HttpsEverywhere;HstsPreload;ForwardedHeaders;HostFiltering;ResponseCaching;ResponseCompression;CORS;HumansTxt;SecurityTxt;RobotsTxt;Swagger;Versioning;DataContractSerializer;StyleCop</DefineConstants>
+    <DefineConstants>$(DefineConstants);TreatWarningsAsErrors;Kestrel;IIS;NGINX;Azure;ApplicationInsights;HealthCheck;HttpsEverywhere;HstsPreload;ForwardedHeaders;HostFiltering;ResponseCaching;ResponseCompression;CORS;HumansTxt;SecurityTxt;RobotsTxt;Swagger;Versioning;DataContractSerializer;StyleCop;Docker</DefineConstants>
     <!-- Workaround to build this project: https://github.com/dotnet/templating/issues/1438 -->
     <ApplicationInsights>true</ApplicationInsights>
     <Azure>true</Azure>
     <Swagger>true</Swagger>
     <Versioning>true</Versioning>
     <StyleCop>true</StyleCop>
+    <Docker>true</Docker>
```

Comment:
What does this do?

Reply:
It's for authoring templates. It's a workaround, so that I can use the #if Docker preprocessor variable.